apps/src/blockly/addons/functionEditor.js

```diff
@@ -347,7 +358,7 @@ export default class FunctionEditor {
     const returnValue = {
       ...blockConfig,
       x: 20,
-      y: 40,
+      y: 75,
```

Comment:
nit: now that we have a constant for the header height, can we use that here too (plus whatever buffer we are using)?

Reply:
Updated. There is still a bit of magic number buffer in each direction, unfortunately.